tests/e2e/stateapp/stateapp_test.go

```diff
@@ -0,0 +1,297 @@
+// +build e2e
+
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package stateapp_e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/dapr/dapr/tests/e2e/utils"
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+	"github.com/dapr/dapr/tests/runner"
+	guuid "github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+const appName = "stateapp"     // App name in Dapr.
+const healthChecks = 3         // Number of get calls before starting tests.
```

Comment:
Consider naming `numHealthChecks`.  `heathChecks` makes it sound like the type of health checks to perform.

Reply:
Fixed.